android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/configuration/action/multi/WakeUpAction.kt

```diff
@@ -21,13 +20,24 @@ internal class WakeUpAction<C : Parcelable> : MultiConfigurationAction<C> {
      */
     override fun execute(
         state: WorkingState<C>,
-        params: ActionExecutionParams<C>
-    ): Map<ConfigurationKey, ConfigurationContext.Resolved<C>> =
-        state.pool.invokeOn(SLEEPING, params) { foundByFilter ->
-            val action = ActivateAction(foundByFilter, params, false)
+        params: TransactionExecutionParams<C>
+    ) {
+        state.pool.invokeOn(SLEEPING, params) { key, foundByFilter ->
```

Comment:
maybe just `key, configuration`? to mention the type of second parameter in its name

Reply:
Sure. It's not a `Configuration` though but a `ConfigurationContext` (think Resolved / Unresolved), which itself should have a better name.